text/src/autogluon/text/automm/configs/model/fusion_mlp_image_text_tabular.yaml

```diff
@@ -0,0 +1,77 @@
+model:
```

Comment:
general question on configs: What happens if I typo one of the fields (such as `hidden_size_INVALID` instead of `hidden_size`. Does the code error / warn me, or does it silently ignore the value and train anyways?

Reply:
It will throw errors when the required keys are missing. Adding unused keys has no effect.